examples/bunnymark/main.rs

```diff
@@ -1,6 +1,7 @@
 #![allow(irrefutable_let_patterns)]
 
 use blade_graphics as gpu;
+use blade_graphics::Memory;
```

Comment:
please follow the existing style everywhere!
all the `blade_graphics` stuff is accessed as `gpu::` here

Reply:
Yhea my bad, i tried doing that, but it seems i accidentally had my editor import still some. I should probably turn that off.😅

Reply:
Still there?